README.md

```diff
@@ -744,6 +744,63 @@ class CustomFactory(ModelFactory[Any]):
 Where `cls._get_faker()` is a `faker` instance that you can use to build your
 returned value.
 
+## Partial parameters random generation
```

Comment:
also update the TOC at the beginning of the readme pls

Reply:
Done.